ext/node_resolver/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+# Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
+
+[package]
+name = "node_resolver"
```

Comment:
Make sure to claim this crate

Reply:
Looks like it's already claimed by us (https://crates.io/crates/node_resolver). Bump version do 0.2 then?

Reply:
It will be bumped to 0.2 on the next release.